cl/recap/factories.py

```diff
@@ -14,3 +14,12 @@ class Meta:
eb\xa7Ta\x9b\x97\x0e\xc0\xc6\x8c\x17"7HgC^^\x01v\xff\xcc0C\\\x8f\x1a\xb7=\xab\xcbpx\xe9a\x82m2\x0ei\xe2\xa7\xcc\xcb\x11\xc0\xe6\x8c?\x99"\x82"\xe6G\x18\x8a/\xc4I\xb6\xbe\xd9\x8f{\xa9\xc9\xaf\xfd\x8f\x93\xcfN\xa7\xfbl\xd9\xef\xdf\xeb\xda\xc5|\xf7\xe7\xc3\x89\x03\x9f2Ey\x12\n\xd7%\xcbc,\xa6\rY&\xe8N\xba\xf1\x997u\xf8\xc9\xc4\x1dG\x0cGL\xe2\xc6\xe4\x101\xcfG\xd4\xb9\x8fpDe\x1a\xa2 \xe0\x18e\x98\xc8\xf1p$\xf1\xbc.5~\xd7\xa4+\xfe\xd59\xc1\xd8\xc9\x89W\xd6\xb2\xdf\x11\x84\xcc\xe0B\xce\xf0\xc3DV\x94C\xad\xa9\xab\x86\x8a\xea\x8a\xdb\xec\xa3\x11"~\xd3T\x8fk\xaaV7E\xc3\xd0\r\xb1\xdaV5\xb1\xde\xd2\x0fDS\xb7t\xb3j\xb7k\x86V\x85\x1fPK\x07\x08\x88\xba\xfc\xf1b\x01\x00\x00R\x02\x00\x00PK\x01\x02\x14\x03\x14\x00\x00\x00\x00\x00+c\xa4T\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x12\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xedA\x00\x00\x00\x001-20-cv-10189-FDS/UT\r\x00\x07\xf3\xa8rb\xf5\xa8rb/\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00+c\xa4T\x96\xde\x02\xa8m\x00\x00\x00\xd2\x00\x00\x00\x1c\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81P\x00\x00\x00__MACOSX/._1-20-cv-10189-FDSUT\r\x00\x07\xf3\xa8rb\xf5\xa8rb4\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00Gc\xa4TU5K\x14G\x01\x00\x00\x04\x18\x00\x00\x1b\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\'\x01\x00\x001-20-cv-10189-FDS/.DS_StoreUT\r\x00\x07&\xa9rb&\xa9rb&\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00Gc\xa4T\x0b\x88\xc085\x00\x00\x00x\x00\x00\x00&\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\xd7\x02\x00\x00__MACOSX/1-20-cv-10189-FDS/._.DS_StoreUT\r\x00\x07&\xa9rb4\xa9rb4\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x008c\xa4T\xd5\xe9\x03\xd7U\x00\x00\x00\x82\x00\x00\x00\x1a\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\x80\x03\x00\x001-20-cv-10189-FDS/12-1.pdfUT\r\x00\x07\x0c\xa9rb\x0c\xa9rb-\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x008c\xa4T\x88\xba\xfc\xf1b\x01\x00\x00R\x02\x00\x00%\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81=\x04\x00\x00__MACOSX/1-20-cv-10189-FDS/._12-1.pdfUT\r\x00\x07\x0c\xa9rb\x0c\xa9rb4\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x008c\xa4T\xd5\xe9\x03\xd7U\x00\x00\x00\x82\x00\x00\x00\x1d\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\x12\x06\x00\x001-20-cv-10189-FDS/12-main.pdfUT\r\x00\x07\x0c\xa9rb\x0c\xa9rb/\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x008c\xa4T\x88\xba\xfc\xf1b\x01\x00\x00R\x02\x00\x00(\x00 \x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\xd2\x06\x00\x00__MACOSX/1-20-cv-10189-FDS/._12-main.pdfUT\r\x00\x07\x0c\xa9rb\x0c\xa9rb4\xa9rbux\x0b\x00\x01\x04\xf5\x01\x00\x00\x04\x14\x00\x00\x00PK\x05\x06\x00\x00\x00\x00\x08\x00\x08\x00c\x03\x00\x00\xaa\x08\x00\x00\x00\x00'
```

Comment:
Let's get this into tests/fixtures.py, right? 

And let's rename that file as binary_fixtures.py, maybe or something that indicates what's in there with more specificity?

Reply:
I think... its not being used actually.